files/csgitbot/bot.service

```
@@ -0,0 +1,15 @@
+[Unit]
+Description=Informabot
```

Mi sembrava di averlo scritto prima ma evidentemente me lo sono perso, c'è questo nome da cambiare
```suggestion
Description=CSGitbot
```